.Rbuildignore

```diff
@@ -1,4 +1,4 @@
-^.*\.Rproj$
```

Comment:
Doesn't RStudio modify this when you re-open?

Reply:
It does not for me. Does it for you?

Reply:
Hmmm, I could've sworn it did in the past, but not this time!

Reply:
I have even checked with a full "new Project (that is a package) from GitHub" workflow. And RStudio doesn't modify this in `.Rbuildignore`

I've come to the conclusion that the problem of the missing template was self-inflicted. Although maybe that `.Rbuildignore` line was originally inspired by RStudio?

Reply:
Agreed x2